### ISSUE TITLE
Auto-Sieve still can void items when it has full output

### ISSUE THREAD

**Issue by Hakameda** [Quantum64/ExGregilo#19]
Weirdly, Clicking anywhere on the GUI will void a gravel, or adding and removing the gravel while the output is full - https://gyazo.com/9440ad193eaf4d6d7edf580b5b862bfe

Posted gif on gregtech discord too, Loads a little better

**Comment by Quantum64** [comment]
Does it still do this in the latest build?

**Comment by Hakameda** [comment]
I'll check after work

**Comment by WSpadaro** [comment]
I have tested and the device seems to consume its input even if it does not process when the machine is updated. So if you put a stack of 64 gravel in without any slots left in the output, it will consume one and nothing will happen and it will go down to 63. Also, the machine updates every so often and it will also consume a single input whether or not it can run. My stack of 64 is very slowly disappearing one every 20 seconds or so?

**Comment by Hakameda** [comment]
Yup, can also confirm not working - https://gyazo.com/7c9a1b027f77e3be3d2b735890ab932f

Every time i click it removes a gravel while the output is full, Interestingly, When its down to 1 gravel. The tooltip mod shows 0 inside the machine but it isn't a ghost item - http://prntscr.com/hohdyk
